js/script.js

```diff
@@ -316,7 +316,11 @@ function stop_teleprompter()
 // Update Teleprompter
 function update_teleprompter()
```

Comment:
Probably need to update this to `update_teleprompter(evt)` otherwise the `evt` used inside this function will be undefined